### PR TITLE
Release v1.4.0/v0.29.0

### PR DESCRIPTION
**Pull request by MrAlias**
- Update `otel` dependencies
- Drop exact support in datadog and dogstatsd exporters
- Drop support for histograms in datadog exporter
- Deprecate the datadog, cortext, and dogstatsd exporters
- Update all `otel/sdk/export/metric*` package to use the non-deprecated replacement.
- Bump versions
- Update all `version.go` files (looks like this was missed last release).

### Added

- Add `WithClientTrace` option to `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp`. (#875)

### Changed

- All metric instruments from the `go.opentelemetry.io/contrib/instrumentation/runtime` package have been renamed from `runtime.go.*` to `process.runtime.go.*` so as to comply with OpenTelemetry semantic conventions. (#1549)

### Fixed

- Change the `http-server-duration` instrument in `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp` to record milliseconds instead of microseconds.
  This changes fixes the code to comply with the OpenTelemetry specification. (#1414, #1537)
- The `"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp".Transport` type now correctly handles protocol switching responses.
  The returned response body implements the `io.ReadWriteCloser` interface if the underlying one does.
  This ensures that protocol switching requests receive a response body that they can write to. (#1329, #1628)

### Deprecated

- The `go.opentelemetry.io/contrib/exporters/metric/datadog` module is deprecated. (#1639)
- The `go.opentelemetry.io/contrib/exporters/metric/dogstatsd` module is deprecated. (#1639)
- The `go.opentelemetry.io/contrib/exporters/metric/cortex` module is deprecated.
  Use the go.opentelemetry.io/otel/exporters/otlp/otlpmetric exporter as a replacement to send data to a collector which can then export with its PRW exporter. (#1639)

### Removed

- Remove the `MinMaxSumCount` from cortex and datadog exporter. (#1554)
- The `go.opentelemetry.io/contrib/exporters/metric/dogstatsd` exporter no longer support exporting histogram or exact data points. (#1639)
- The `go.opentelemetry.io/contrib/exporters/metric/datadog` exporter no longer support exporting exact data points. (#1639)

### CODE REVIEW

File: versions.yaml [hunk 1]
```diff
@@ -82,14 +82,17 @@ module-sets:
...
   experimental-metrics:
-    version: v0.28.0
+    version: v0.29.0
+    modules:
+      - go.opentelemetry.io/contrib/instrumentation/host
+      - go.opentelemetry.io/contrib/instrumentation/host/example
+      - go.opentelemetry.io/contrib/instrumentation/runtime
+      - go.opentelemetry.io/contrib/instrumentation/runtime/example
+  experimental-metric-exporters:
+    version: v0.29.0
```
Comment (Aneurysm9) [inline versions.yaml:92]: I think we should deprecate all of these exporters.  `dogstatsd` can only support counters and gauges and `cortex` is probably better served by shipping OTLP to a collector and using its PRW exporter.  Deprecating them now will allow us to excise them by the time we start landing significant metrics API/SDK changes.

Comment (MrAlias) [reply versions.yaml:92]: Sure, will do.

**Review by Aneurysm9 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,24 +8,36 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+## [1.4.0/0.29.0] - 2022-02-11
```
Comment (pellared) [inline CHANGELOG.md]: needs to be updated of course 😉

Comment (MrAlias) [inline CHANGELOG.md:11]:
```suggestion
## [1.4.0/0.29.0] - 2022-02-14
```

**Review by pellared — APPROVED** [review]
Please double-check if the following commit/PR should be included in CHANGELOG:
- https://github.com/open-telemetry/opentelemetry-go-contrib/pull/1546

File: CHANGELOG.md (hunk 2 above)
Comment (pellared) [reply CHANGELOG.md]: @MrAlias You are fast!

Comment (MrAlias) [reply CHANGELOG.md]: I've been working on my precognition skills :laughing:

**Comment by MrAlias** [comment]
> Please double-check if the following commit/PR should be included in CHANGELOG:
> 
> * #1546

I'll update the changelog to include this change. Thanks for the catch.

**Pull request merged**